logs.go

```diff
@@ -73,6 +73,20 @@ func (d *driverWithLogs) NewWindow(c WindowConfig) Window {
 	return &windowWithLogs{Window: w}
 }
 
+func (d *driverWithLogs) NewController(c ControllerConfig) Controller {
```

Comment:
Please keek the same func order as in driver. I think its after newcontextmenu

Reply:
done 👍